orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/lambda/lambdaFunctionTask.java

```diff
@@ -27,15 +27,22 @@
 import java.util.HashMap;
 import java.util.Map;
 import javax.annotation.Nonnull;
+import org.apache.commons.lang3.StringUtils;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
+/*
+  lambdaFunctionTask creates a Clouddriver operation using the field labeled
+  'operation'. The context of the task are added to the operation body. Users
+  will have to know what Clouddriver is expecting to use this task. It
+  essentially just passes an arbitrary operation over to Clouddriver.
+*/
 @Component
-public class updateLambdaFunctionTask extends AbstractCloudProviderAwareTask implements Task {
+public class lambdaFunctionTask extends AbstractCloudProviderAwareTask implements Task {
 
   @Autowired KatoService katoService;
 
-  public static final String TASK_NAME = "updateLambdaFunctionConfiguration";
+  public static final String TASK_NAME = "lambdaFunction";
```

Comment:
Is `TASK_NAME` used anywhere?

Reply:
Yeah, it's used in the stage [here](https://github.com/spinnaker/orca/pull/2943/files#diff-d93c048a1464581cd7051d6b3ab58794R33)